webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2480,19 +2480,88 @@ var CURRENT_GROUP_DATA={};
                     _this.toggleMediaOptions(this);
 
                 });
-                $mck_text_box.keyup(function (){
+             
+            _this.cursorPosition =  function () {
+                var sel = document.getSelection();
+                sel.modify("extend", "backward", "documentboundary");
```

Comment:
@hardikparashar `.modify()` will not work in Internet Explorer and MS Edge. Need to find some other solution. https://developer.mozilla.org/en-US/docs/Web/API/Selection/modify